### PR TITLE
Go rewrite

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Rewrite server in Go. 

Thanks, @diamondburned!

### CODE REVIEW

File: index.html
Comment (EthanThatOneKid) [inline index.html:1]: Move this to a dedicated example directory.

File: server/server.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/diamondburned/arikawa/v3/api/webhook"
+	"github.com/diamondburned/arikawa/v3/discord"
+	"github.com/diamondburned/arikawa/v3/utils/sendpart"
```
Comment (EthanThatOneKid) [inline server/server.go:12]: Use `libdb.so` over `github.com/diamondburned`.

Comment (diamondburned) [reply server/server.go:12]: This won't work. See https://github.com/diamondburned/arikawa/blob/v3/go.mod#L1.

Comment (EthanThatOneKid) [reply server/server.go:12]: > This won't work. See https://github.com/diamondburned/arikawa/blob/v3/go.mod#L1.

Thanks for the heads up!

File: index.html
Comment (EthanThatOneKid) [reply index.html:1]: Resolved in <https://github.com/EthanThatOneKid/discord_uploader/pull/1/commits/652bd9faefb8bd8f48acb471bf8e06b2fb6732f8>.

**Comment by EthanThatOneKid** [comment]
Testing the Netlify deployment, I encountered the following error:

```
index.html:1 Access to fetch at 'https://deploy-preview-1--deluxe-manatee-ee476f.netlify.app/' from origin 'http://127.0.0.1:5500' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

To resolve this error message, the appropriate HTTP headers must be sent back in the server's response.

**Pull request merged**